src/test_fixture/sharing.rs

```diff
@@ -170,12 +170,34 @@ impl<F: Field> Reconstruct<F> for [&Replicated<F>; 3] {
     }
 }
 
+impl<F: Field> Reconstruct<F> for [&MaliciousReplicated<F>; 3] {
```

Comment:
You shouldn't need this function; the sort returns the validator and so you can validate to downgrade the malicious shares that you have.

Reply:
True.. Changed this